### DISCUSSION TITLE
建议基本田激活YL采用加强参考值的稳定分流机制

### DISCUSSION THREAD

**Discussion by yancanlou** [famland/website#67] [Ideas]
目前新基本田的YL是完全流入资金流动池，此种情况会出现：
当出现YL价格偏离参考值时（比如当前价格被打到0.5，而参考价为0.8时），激活基本田做市是在加强YL价格（0.5）的偏离，与稳定理念不符，变成适应价格的助涨助跌，产生不稳定。

建议采用加强参考值的稳定分流机制：
激活基本田做市时，以按实时市场价格做市新激活投入YL量不变。然后根据以参考价为标准，将参考价对应的YL注入流动池内，分流额由蓄水池控制。
流动池YL注入量=实时价格*激活投入YL量/参考价格=250/参考价格
实时价格<参考价格时，分配是：蓄水池新增量或注销量=激活投入YL量-流动池YL注入量    
例：价格是0.5，参考价为0.8时，分流到蓄水池或注销量=500-312.5=187.5
实时价格>参考价格时，分配是：蓄水池减少量或新发行量=流动池YL注入量-激活投入YL量
例：价格是1，参考价为0.8时，蓄水池减少量或新发行量=312.5-250=62.5

**Comment by happybig** [comment]
建议不错，值得深究。

**Reply by mjy368837** [reply]
多多建议，真的不错，

**Comment by itey** [comment]
建议不错！从势能效应来看，由下向上难，由上向下易，特别是YL价格在基准价格以下时，通过每次做市就可以给YL的价格提供一个向上拉伸趋势和力量。这种力量在共享农场机制的作用下，玩家通过日后不断复投基本田或野田产生的效应会更加明显。

**Reply by yancanlou** [reply]
谢谢认可！

**Comment by otto-001** [comment]
非常不错的建议。做市基本田可以助力价格上涨。

**Comment by cryptosoul-vioiv** [comment]
好建议！如此，则新增做市资金是在增长YL价格上涨的势能，再有利于维持某个稳定价，以此达到稳定币的目的。
